typescript/submissionEvents.d.ts

```diff
@@ -102,6 +102,7 @@ export type TrimSubmissionEvent = FormSubmissionEventConditional & {
     groupFiles?: boolean
     usePagesAsBreaks?: boolean
     excludedElementIds?: string[]
+    preventExtensionlessAttachments: boolean
```

Comment:
This will need to be optional, (as you've put in the docs), because all previously saved configurations will not have this property